packages/@icon-magic/distribute/src/create-icon-template.ts

```diff
@@ -25,9 +27,13 @@ export async function createHbs(
     const xml = doc.parseFromString(contents as string, 'image/svg+xml');
     const el = xml.documentElement;
     const id = el.getAttributeNode('id');
-    const iconName = id ? id.value : '';
+    let iconName = id ? id.value : '';
     // Strip id
     el.removeAttribute('id');
+    // Remove the "-mixed" suffix from the name. File will have same name as light version.
+    if (!doNotRemoveSuffix) {
+      iconName = iconName.replace(/-mixed$/, '');
```

Comment:
is it possible to check the `colorScheme` here? just to make sure we're not removing `mixed` if it just happens to be a part of the name.

Reply:
Sure we can do that, although that regex makes sure that it ends exactly with `-mixed`.  "mixed" can be anywhere in the name(including the end) as long as the name does not END with dash+mixed.

Reply:
done :)